docs/docs/gatsby-lifecycle-apis.md

```diff
@@ -15,6 +17,10 @@ Gatsby's design principles include:
 
 # High level Overview
 
+The following model gives a conceptual overview of how data is source and transformed in a Gatsby site:
+
+<LayerModel />
```

Comment:
If there isn't an initialView specified, it would help to default to the first tab since the component otherwise looks like it relates to the text content below.

Reply:
@marcysutton yes. I like the interactive tab switcher. It is a new component for the docs. 

But I see there is a problem that nobody knows that I can tap on the icons and see more conten. I don't know how it could be done better. Maybe a hint "Klick on the icons and switch the datasource" or something like this.

I see also other elements like this cycle through the layers ever xx seconds to show there is more content 

Also it would be nice to have all 5 options all visible for printing or something (like on some pages you can have a gallery and also.the option "view all on one page)

Reply:
I would advocate against auto-rotating, personally – it would distract from the reading experience and cause accessibility problems. If the icons don't look interactive enough, perhaps there's a visual design tweak we could make. Having to say "click these icons" suggests the design could be improved -- perhaps making the buttons look more like buttons with edges/borders could help, even just having one selected by default could help.

I agree about showing all content for a the print stylesheet -- that's a great point, and if it could be handled with CSS now, it would make future work to print the docs go a little bit faster.

Reply:
I think defaulting to the initial view will help with the issue that they may not look clickable, that's great feedback 👍